modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -81,7 +81,7 @@ Download the latest https://owncloud.org/download/#owncloud-server-tar-ball[ownC
 [source, console]
 ----
 cd /var/www/
-wget https://download.owncloud.org/community/owncloud-10.2.1.tar.bz2
+wget https://download.owncloud.org/community/owncloud-10.3.2.tar.bz2
```

Comment:
This should use the available asciidoc variables

Reply:
agree

Reply:
I submitted a commit to implement this change.